easybuild/easyconfigs/d/dask/dask-0.20.0-foss-2018b-Python-3.6.6.eb

```diff
@@ -0,0 +1,90 @@
+easyblock = 'Bundle'
+
+name = 'dask'
+version = '0.20.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://github.com/dask/dask/'
+description = """Dask provides multi-core execution on larger-than-memory datasets using blocked algorithms
+ and task scheduling."""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+
+dependencies = [
+    ('Python', '3.6.6'),
+    ('PyYAML', '3.13', versionsuffix),
+]
+
+exts_defaultclass = 'PythonPackage'
+exts_default_options = {
+    'download_dep_fail': True,
+    'use_pip': True,
+}
```

Comment:
All the `exts_*` variables above are not needed with `PythonBundle`.
However, `use_pip = True` is still necessary.